doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
I don't feel strongly.  I'm just suggesting a way to make the examples a lot easier to read. Embedding them in doctests adds lots of clutter.  Obviously, the buildout docs were an extreme example.  If these examples aren't tested now, then why make them doctests?  Or why not separate the configuration text from the text that loads it?

If the examples are tested now, converting the tests to Manuel is very easy. Manuel would only need to be a testing dependency.

In my experience, untested examples are often wrong. :)

But this is *only* a suggestion.